src/ssl.c

```diff
@@ -985,8 +985,10 @@ int wolfSSL_SetTmpDH(WOLFSSL* ssl, const unsigned char* p, int pSz,
     if (pSz < ssl->options.minDhKeySz)
         return DH_KEY_SIZE_E;
 
+    #ifndef WOLFSSL_WPAS
```

Comment:
In the wolfSSL code, currently ownership of dh params is by client side or server side with weOwn flag.  This breaks that logic.  A wolfSSL object is client or server or v23 generic which also "picks" a side by default.